### ISSUE TITLE
Work with GeneCards/MalaCards to link to Open Targets and vice versa

### ISSUE THREAD

**Issue by MichaelaEBI** [opentargets/platform#541]
GeneCardsSuite (Weizmann Institute of Science) want to link out to Open Targets from their GeneCards and MalaCards resources. In reverse, Open Targets will link to their target and disease pages.

**GeneCards links to Open Targets:**
1. Link to the Open Targets target profile page in 'Additional Gene Information' within the Gene summary section.
- [ ] Provide a tsv file with Ensembl IDs and HGNC names (maybe UniProt) with each OT release --> Automate generation of such a file
2. In the Disorder section, a.) add Open Targets to the list of sources of disease information and b.) add superscript links in the table
 - [ ] Question: Find out what a.) means, ie., would Open Targets be listed under 'Additional Disease Information for gene x'? --> Link to target association page
 - [ ] Question: Find out whether they want target-disease associations from Open Targets or whether they would simply link to Open Targets evidence pages for target-disease associations they already have in their website. Where do their target-associations come from (are they curated)?

**MalaCards links to Open Targets:**
1. Include target-disease associations from Open Targets in the MalaCards gene table (e.g. https://www.malacards.org/card/prostate_cancer#related_genes), with links to the Open Targets evidence page.
 - [ ] Question: Do they a.) want to use Open Targets as an additional supporting reference for the targets they have identified for a disease or b.)would they want all of our associated genes (should a cut-off be applied to the association score)? Give them an idea for how many targets we typically associate with a disease.
 - [ ] If their choice is a.), we would have to figure out the best way to provide information for disease targets they have identified.
2. Link to the Open Targets disease page (e.g. https://www.targetvalidation.org/disease/EFO_0001663) from the MalaCards ‘Aliases’ section (e.g.https://www.malacards.org/card/prostate_cancer#aliases_descriptions)
 - [ ] Need to map EFO3 to their disease identifier(s). [with @afaulconbridge]
 - [ ] Ask them to send us the list of their diseases with all IDs.

**Links from Open Targets to GeneCards:**
 - [ ] Decide where on the Target profile page we would link to other resources like GeneCards (Genetics portal links to Ensembl, HGNC, UniProt and others). [with @andrewhercules  & @mirandaio]
 - [ ] We need a list of their targets, ideally Ensembl IDs.

**Links from Open Targets to MalaCards:**
 - [ ] Decide whether to link to other resources other than MalaCards (OMIM for Mendelian diseases) and where this should be done. [with @andrewhercules & @LucaFumis ]
 - [ ] Need a list of their diseases (see above).

**Comment by andrewhercules** [comment]
@MichaelaEBI and @deniseOme, I'm happy to update the spec for the target profile page rewrite to include links to GeneCards and MalaCards. We could integrate the links alongside the Ensembl and UniProt links?

If you think this is suitable, I can update [the target profile block design spec](https://github.com/opentargets/platform/issues/335) to include a link to GeneCards. This would be similar to [the design for the Genetics Portal gene page](https://github.com/opentargets/genetics/issues/303), which includes a link-out to Gene Cards (and other sites) directly underneath the HGNC symbol.

**Comment by deniseOme** [comment]
Current version of new target profile page shows GeneCards among others (https://platform-app.netlify.com/target/ENSG00000091831)

![Screen Shot 2019-06-07 at 10 20 10](https://user-images.githubusercontent.com/6472381/59094390-f173f780-890d-11e9-801a-f2da49adb893.png)

@d0choa @mirandaio We now need to consider adding the MalaCards ID to the Disease profile page in the Platform:
Malacards: https://www.malacards.org/card/asthma
Platform: https://www.targetvalidation.org/disease/EFO_0000270

one we are happy with the MalaCards --> EFO mappings provided by SPOT (thanks to Zoe):
https://docs.google.com/spreadsheets/d/1GSOkYegyQOdz21gdhMEex6gkOsHzlF9DWApq096FIo8/edit?usp=sharing

**Comment by d0choa** [comment]
Thanks @deniseOme. We're expecting to have a version of the Target Disease Page for the first week of August. I'm tagging this issue accordingly.  I'll talk with @MichaelaEBI to make sure we're happy with the mappings and we get regular updates.

**Comment by deniseOme** [comment]
@MichaelaEBI and I had a meeting this morning re. the disease mapping sent from MalaCards. We are currently double checking some info with them as it seems they now have their diseases in EFO. More to follow.

**Comment by d0choa** [comment]
Do we have any update on the malacards mappings?

**Comment by deniseOme** [comment]
@d0choa, this is the summary from our last email with GeneCards/MalaCards on 19/06/19. 

- they are happy with the target profile page prototype we've sent them with the new GeneCards link (as depicted in the above image). We should let them know when the new TPP is live.

- they are aware of the redesign of the disease profile page. Once we have some prototypes for this, we can share these with them.

- they now have disease mappings to EFO. Their mapping file is attached here. 
[Uploading MalaCards_diseases_EFO_identifiers_v4.10.txt…]()

Out of 19,900 MalaCards diseases, only 2,232 have EFO IDs.

- some but not all MalaCards diseases have ICD10 codes. Those with ICD codes come from DO and OrphaNet, resources where MalaCards get ICD codes from.

- the way they generate their URLs is like `https://www.malacards.org/card/diseaseslug/` 
e.g. https://www.malacards.org/card/asthma

- they were given the 19.04 gene mapping tsv file generated by @MichaelaEBI and were supposed to have a look and get back to us. 

What is next?

- We don't have a plan yet AFAIK on how to provide the disease and gene mapping files to them whenever we have new release. 

UniProt and Reactome already link out to us, so we can follow the footsteps of these resources. Not sure if @MichaelaEBI gives a new file every new release to Reactome.

- Work out how to do the disease mapping for those MalaCards diseases with no ICD or EFO codes.

- Share the new disease profile page prototype when ready.

@MichaelaEBI, anything I have forgotten to report here?

@d0choa you may know already that we have a file from Zoe with the ICD-->EFO3 mappings. Not sure how we want to proceed with the disease mapping (e.g. to wait or not to wait for when we have the EFO3 slim?).

**Comment by MichaelaEBI** [comment]
@deniseOme, yes I provided a file with all targets in Open Targets to reactome for each of their releases. This will now become easier when we provide a target list with each Open Targets release (see ticket #657).

**Comment by d0choa** [comment]
From our side I consider this completed. MalaCards link-outs remain pending as we have no way to know the landing urls.

We have done our best to provide the lists of targets and diseases, so that others can link to us.

**Comment by deniseOme** [comment]
With the release of the [alpha version](https://alpha.targetvalidation.org/) our link outs to GeneCards are up and running. I've contacted GeneCards for an update from our front and heard that the links in GeneCards to Open Targets Platform in their Summaries and Disorders sections have been implemented.

![openTargetsDisorders](https://user-images.githubusercontent.com/6472381/85141837-51022400-b23f-11ea-92f9-1194eb6ca254.PNG)
![openTargetsSummarySection](https://user-images.githubusercontent.com/6472381/85141843-53647e00-b23f-11ea-82b9-b46b048191ad.PNG)

This will go live in their version 5.0 in a month or two. They will also include this great news in their "What’s New" page for 5.0.
